docs/install/knative-with-operators.md

```diff
@@ -0,0 +1,244 @@
+and an [Eventing operator](https://github.com/knative/eventing-operator), and you can choose to install one or both independently.
+
+## Prerequisites:
+
+### Set up a Kubernetes cluster
+
+Knative operators are still in Alpha phase. They focus on installation on a generic Kubernetes platform, such as
+Docker Desktop, Minikube, or kubeadm clusters, and the operators do not perform any platform specific customization.
+If you are not sure the customization of the Knative operators for vendor-specific platforms, use the generic Kubernetes
+platform.
+
+If you have only one node for your cluster, set CPUs to at least 6, Memory to at least 6.0 GB, Disk storage to at
+least 30 GB. If you have multiple nodes for your cluster, set CPUs to at least 2, Memory to at least 4.0 GB, Disk storage
+to at least 20 GB for each node.
+
+You need to make sure that your Kubernetes cluster is able to access the internet, since Knative operators download
+images online.
+
+Vendor-specific Kubernetes services, such as Google Kubernetes Engine, IBM Kubernetes Service, or OpenShift, may need
+some additional configurations, after the Knative is installed using the operators. In this tutorial, we do not introduce
+the customization or additional configurations, that may vary from platform to platform.
+
+### Install Istio
+
+Knative Serving needs an ingress or gateway to route inbound network traffic to the services. Knative Operators only support
+Istio in this release. Since we talk about the installation on generic Kubernetes service, Istio can be installed with
+the following steps:
+
+1. [Download and install Istio](https://knative.dev/development/install/installing-istio/#downloading-istio-and-installing-crds). Go through all the 4 sub-steps.
+2. [Update your Istio to use cluster local gateway](https://knative.dev/development/install/installing-istio/#updating-your-install-to-use-cluster-local-gateway).
+
+## Limitations of Knative Operators:
+
+Knative operators use custom resources as the sources of truth to configure your Knative. The current custom resources
+still have limited capabilities, which can not configure everything to meet Knative needs, but Knative operators are
+making progress in extending their capabilities. One of the major missing points is that Knative operators do not currently
+provide HA capabilities.
+
+The open source versions of Knative operators have not been tested on any existing production environment, so use
+them for development or test purpose. However, we encourage you to experiment Knative operators on top of the Kubernetes
+cluster for your production environment, helping us find out what we can potentially improve in Knative operators by
+reporting your issues.
+
+## Install Knative Serving with Operator:
+
+All the releases for Knative Serving Operator can be found on [this page](https://github.com/knative/serving-operator/releases).
```

Comment:
Change to:
"Information about Knative Serving Operator releases can be found on the [Releases](https://github.com/knative/serving-operator/releases) page."